content/blog/entries/cc-search-extension-wrapping-up-gsoc-2020/contents.lr

```diff
@@ -0,0 +1,171 @@
+
+<figure >
+    <img src="old-extension-filters.gif" style="width: 70%">
+    <figcaption>
+        <em>Filters in the old version</em>
+    </figcaption>
+</figure>
+
+<figure>
+    <img src="new-extension-filters.gif" style="width: 70%">
+    <figcaption>
+        <em>Filters in the new version</em>
+    </figcaption>
+</figure>
+
+_Rationale_: This will allow users to be more precise in their queries when searching.
+
+### Browsing by source
+The extension now has a "sources" section. The latest sources are fetched from the `/sources` endpoint. Clicking a source link triggers a request to the `/image` endpoint to get the images associated with it.
+
+<figure >
+    <img src="source-section-light.gif" style="width: 70%">
+    <figcaption>
+        <em>Source section</em>
+    </figcaption>
+</figure>
+
+<figure>
+    <img src="source-section-dark.gif" style="width: 70%">
+    <figcaption>
+        <em>Source section in dark mode</em>
+    </figcaption>
+</figure>
+
+_Rationale_: This opens an avenue for exploration of all the different sources which are available in the catalog. This is advantageous for the users who are not familiar with the type of content a particular source provides. They might run into a source that has a huge catalog of high-quality images that they are looking for.
+
+### Search by tags
+Most of the images have some tags associated with them, which are also sent along with the image data by the API. This, and the flexibility of the `/image` endpoint, paved the way for the addition of searching for images using image-tags.
+
+<figure >
+    <img src="search-by-image-tag.gif" style="width: 70%">
+    <figcaption>
+        <em>Search by image tag</em>
+    </figcaption>
+</figure>
+
+_Rational_ - Image tags will allow users to incrementally make their queries better and more specific.
+
+### Related images
+In the image detail section of any particular image, you can now see several recommendations. This has been made possible by adding support for the [`/recommendations/images/{identifier}`](https://api.creativecommons.engineering/v1/#tag/recommendations) endpoint of the API.
+
+<figure >
+    <img src="related-images.gif" style="width: 70%">
+    <figcaption>
+        <em>Image recommendations</em>
+    </figcaption>
+</figure>
+
+_Rationale_ -  This will help users find a variety of images that fit their requirements and also explore the images that would not usually show up on the initial pages of the search result. 
+
+## Improvements to bookmarks section 
+The bookmark section has great prominence in CC Search Extension because the export/import workflow is tied to it and unlike the search result data, the bookmarks data is preserved across user sessions (closing the extension does not wipe out the bookmarks).  It has undergone some crucial improvements like caching, voluntary loading and increase in  the number bookmarks that it can hold (the limit now is 300 which earlier was ~50)
```

Comment:
```suggestion
The bookmark section has great prominence in CC Search Extension because the export/import workflow is tied to it and unlike the search result data, the bookmarks data is preserved across user sessions (closing the extension does not wipe out the bookmarks).  It has undergone some crucial improvements like caching, voluntary loading and increase in the number bookmarks that it can hold (the limit now is 300 which earlier was ~50).
```